spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnSelector.scala

```diff
@@ -28,7 +28,10 @@ case class SomeColumns(columns: ColumnRef*) extends ColumnSelector {
   }.toMap
 
   override def selectFrom(table: TableDef): IndexedSeq[ColumnRef] = {
-    val missing = table.missingColumns(columns).filterNot(_ == RowCountRef)
+    val missing = table.missingColumns(columns).filterNot {
+      case _: FunctionCallRef => true
```

Comment:
I completely agree with you and in our use case we do check they are present by including them at the same level than the function reference itself. That is: Our `CassandraSourceRelation` expansion class includes the function call reference and the columns used by it within `maybeSelect`s `requiredColumns` parameters (https://github.com/Stratio/Crossdata/pull/165/files#diff-9a6f82a00d7bc8b98fd3c59ff7310321L156)